Multisig/UI/UI Library/View Controllers/UIViewController+NavigationItem.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UIViewController+NavigationItem.swift
+//  Multisig
+//
+//  Created by Moaaz on 1/22/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+extension UIViewController {
+    func createCloseButton () {
+        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(named: "ico-close-x"), style: .plain, target: self, action: #selector(CloseModal.closeModal))
```

Comment:
maybe it is better to create a static create function to create UIBarButtonItem and then use it in controller to assign it wherever it is needed - or is this only used in the left bar button item everywhere?

Reply:
if it's used everywhere - it's good to refactor it in other places

Reply:
It's only for left item, when we will merge all pieces together I'm going to use this one for all,